pyproject.toml

```diff
@@ -97,10 +97,12 @@ line_length = 88
 known_first_party=["pytest_memray"]
 
 [tool.towncrier]
-package = "pytest-memray"
-package_dir = "src"
-filename = "NEWS.rst"
-directory = "news"
+name = "pytest-memray"
+filename = "docs/news.rst"
```

Comment:
In general we prefer to have the news file at top level so it is easier to find.

Reply:
What do you mean? The .rst is the final rendered document.

Reply:
Not true, the rendered document is the HTML page, e.g. https://pytest-memray--20.org.readthedocs.build/en/20/news.html#